js/stdlib/ts/ALGO.ts

```diff
@@ -1376,3 +1376,5 @@ export const verifyContract = async (ctcInfo: ContractInfo, backend: Backend): P
 
   return true;
 };
+
```

Comment:
This is so the frontend can call 
```javascript
const stdlib = await loadStdlib();
const exports = getExports(stdlib.reachStdlib)
``` 
instead of passing an arbitrary ctc.

Reply:
I think the interface should be `getExport(stdlib)` and it will know to look at `stdlib.reachStdLib`